GameServerLib/Scripting/CSharp/CSharpScriptEngine.cs

```diff
@@ -115,32 +115,54 @@ private List<SyntaxTree> GetInvalidSourceTrees(EmitResult result)
 
         public T GetStaticMethod<T>(string scriptNamespace, string scriptClass, string scriptFunction)
         {
-            if (_scriptAssembly == null) return default(T);
+            if (_scriptAssembly == null || !_scriptAssembly.Any())
```

Comment:
Would prefer to have `!_scriptAssembly.Any()` replaced by `_scriptAssembly.Count <= 0`, will make the intent more clear i think